manifests/o/ovsky/ADB/AFI/2.9.5/ovsky.ADB.AFI.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ovsky.ADB.AFI
+PackageVersion: 2.9.5
+InstallerType: exe
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/ovsky/adb-fastboot-installer/releases/download/v2.9.5/ADB.Installer++.v2.9.5.exe
+  InstallerSha256: 29235CFBAE99AE139BF918157DEF0DAEA4E8D20BD5B4135308AA0362167C8D6C
```

```suggestion
  InstallerSha256: F811658721440E299D8B28A5CF2750D1FAE750B907133ABF73CEFE82E4E2812C
```

(Automated response - build 943.)